sdk/containerregistry/arm-containerregistry/test/containerregistry_examples.spec.ts

```diff
@@ -61,196 +61,136 @@ describe("ContainerRegistry test", () => {
     taskName = "mytaskxxx";
   });
 
-  afterEach(async function () {
+  afterEach(async () => {
     await recorder.stop();
   });
 
-  it("registries create test", async function () {
-    const res = await client.registries.beginCreateAndWait(resourceGroup, registryName, {
-      location: location,
-      tags: {
-        key: "value"
+  it("registries create test", async () => {
+    const res = await client.registries.beginCreateAndWait(
+      resourceGroup,
+      registryName,
+      {
+        location: location,
+        tags: {
+          key: "value",
+        },
+        sku: {
+          name: "Premium",
+        },
+        adminUserEnabled: false,
       },
-      sku: {
-        name: "Premium"
-      },
-      adminUserEnabled: false
-    }, testPollingOptions);
-    assert.equal(res.name, registryName)
+      testPollingOptions,
+    );
+    assert.equal(res.name, registryName);
   });
 
-  // it("importPipelines create test", async function() {
```

Comment:
is it fine to delete these importPipelines tests or should we log an issue to track re-enable them? @qiaozha @kazrael2119